introduction.md

```diff
@@ -3,3 +3,9 @@ My fav path is Javascript
 https://www.codeschool.com/learn/javascript
```

Comment:
Since you're becoming a GitHub master, could you also add "Killing history using git rebase" as one of your deadly skills?